.env.example

```diff
@@ -7,6 +7,9 @@ WP_ENV=development
 WP_HOME=http://example.com
 WP_SITEURL=${WP_HOME}/wp
 
+# Optional variables
```

Comment:
I think the default port would be the most popular. Changing `DB_HOST` more than likely means you're just using a remote host (still with the default port).

Anyway, port has nothing to do with this. We should move `DB_HOST` to be an optional comment because it is literally optional. It has a default fallback value.